pymilvus/client/grpc_handler.py

```diff
@@ -131,15 +133,22 @@ def reset_db_name(self, db_name):
         self._setup_db_interceptor(db_name)
         self._setup_grpc_channel()
 
-    def _setup_authorization_interceptor(self, user, password):
-        if user and password:
+    def _setup_authorization_interceptor(self, user, password, token):
+        keys = []
+        values = []
+        if token:
+            authorization = base64.b64encode(f"{token}".encode('utf-8'))
+            keys.append("authorization")
+            values.append(authorization)
+        elif user and password:
             authorization = base64.b64encode(f"{user}:{password}".encode('utf-8'))
-            key = "authorization"
-            self._authorization_interceptor = interceptor.header_adder_interceptor(key, authorization)
+            keys.append("authorization")
+            values.append(authorization)
+        self._authorization_interceptor = interceptor.header_adder_interceptor(keys, values)
```

Comment:
Please set the interceptor if len(keys) > 0 and len(values) > 0, so that ut won't stuck forever